docs/src/pages/guides/composition/composition.md

```diff
@@ -128,6 +128,7 @@ Here is a demo with [React Router DOM](https://github.com/ReactTraining/react-ro
 You can find the details in the [TypeScript guide](/guides/typescript#usage-of-component-property).
 
 ### Caveat with refs
+
```

Comment:
> Did you know that Prettier supports the markdown format scream.

Was about to say that we should look into that. I always press format by accident until I realized the markdown isn't formatted.

Reply:
It happens to me from to time as well.